assets/style.sass

```diff
@@ -117,3 +117,12 @@ section
     color: inherit
     font-style: italic
     font-weight: $regular-weight
+
+.talk
+  margin-bottom: $grid-height
+
+.talk-name
+  font-weight: $semibold-weight
+
+.presentations
+  margin: 0
```

Comment:
It's strange to remove a style explicitly set on `ul` by [line 32](https://github.com/testdouble/cobbler/blob/conference-talks/assets/style.sass#L32), but I feel it creates a good design by making all the presentations of a talk be closely associated with the talk title